dataline-integrations/singer/bigquery/destination/bigquery-destination.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+case "$1" in
+  -c|--config)        CONFIG_FILE="$2"         ;;
+  --)                shift; break             ;;
+esac
+
+cat $CONFIG_FILE | jq ".credentials_json | fromjson" > credentials.json
+GOOGLE_APPLICATION_CREDENTIALS=credentials.json target-bigquery "$@"
```

Comment:
huh. odd that this needs to be an env variable. It can't be passed in as a file

Reply:
The "standard" use case seems to be to run Singer from an environment that has cloud credentials.